### DISCUSSION TITLE
not able to load vst instrument or fx within audiogidder plugin on any daw

### DISCUSSION THREAD

**Discussion by mikeylikesit0475** [apohl79/audiogridder#699] [Help]
I have a windows 10 server and a Acer Predator laptop. I loaded all my vst plugins and the audiogidder server software on my windows server and the audiogridder plugin on my laptop. It doesnt matter which daw i use. Both the audiogidder audio plugin and instrument plugin keeps trying to load the vst plugin but cant. I selected the vst i wanted within audiogidder, audiogidder then tries to connect but then it disconnects and then reconnects and disconnects in a loop where it never loads the vst plugin. I want to say its trying to pull up but cant. I dont know if something is blocking it because at first i thought it was just the audiogridder instrument plugin having the problem. Now im also experiencing the same issue with audiogidder fx plugin.

**Comment by apohl79** [comment]
can you please follow [these steps](https://audiogridder.com/bug-reports/) to provide a bug report?
